include/osl/relation_list.h

```diff
@@ -93,8 +93,12 @@ typedef struct osl_relation_list * osl_relation_list_p;
 void osl_relation_list_idump(FILE *, osl_relation_list_p, int);
 void osl_relation_list_dump(FILE *, osl_relation_list_p);
 void osl_relation_list_pprint_elts(FILE *, osl_relation_list_p, osl_names_p);
+void osl_relation_list_pprint_elts_n(FILE *, osl_relation_list_p, osl_names_p,
+                                                    int);
```

Comment:
`int` parameter looks misplaced too much to the right.  Did you use tabs somewhere?

Same for a couple of places in other files.